Project.toml

```diff
@@ -22,6 +22,7 @@ ImageContrastAdjustment = "^0"
 ImageCore = ">=0.8.3"
 Polynomials = "^0"
 julia = "^1"
+ReferenceTests = "0.7, 0.8"
```

Comment:
Since `ReferenceTests 0.7`, a new keyword `by` is exported so that we can provide a custom equality test function. For binarization purpose, L0(city-block) distance suits best.